src/lambdas/deleteUser/deleteUser.js

```diff
@@ -0,0 +1,50 @@
+const DynamoDB = require('aws-sdk/clients/dynamodb');
+const db = new DynamoDB.DocumentClient();
+const clone = require('clone');
+const Log = require('@dazn/lambda-powertools-logger');
+const TABLE_NAME = process.env.TABLE_NAME || '';
+
+module.exports.handler = async event => {
+    try {
+        const { id: userId } = event.pathParameters || {};
+        if (!userId) throw 'no user id';
+
+        // get item to delete
+        const originalParams = { TableName: TABLE_NAME, Key: { pk: userId, sk: `USER` } };
+        const doc = await db.get(originalParams).promise();
+        const { Item } = doc;
+        if (!Item) throw 'user does not exist';
```

Comment:
This is a reasonable check.
What are your thoughts on returning 200 rather than throwing an error here?

Reply:
I can see 200 being an appropriate response in this situation as well since, like a successful delete, it is confirming that the user does not exist.